lib/ansible/module_utils/vmware.py

```diff
@@ -213,6 +213,19 @@ def find_host_portgroup_by_name(host, portgroup_name):
     return None
 
 
+def _get_vm_prop(vm, attributes):
+    """Safely get a property or return None"""
+    result = vm
+    for attribute in attributes:
+        try:
+            result = getattr(result, attribute)
+        except AttributeError:
+            return None
+        except IndexError:
+            return None
```

Comment:
Note: this could be written as:
`` python
except (AttirbuteError, IndexError):
    return None
```